### ISSUE TITLE
Custom actions feature request

### ISSUE THREAD

**Issue by artemijan** [peco/peco#534]
I'm using peco for a while and it seems to me a very great tool. Apart from that there is a way to improve it dramatically.
Currently there are available only built-in actions and those actions could be called with hotkeys.
It would be great to have a separate section in the config file to setup hotkey to execute shortcut action with selected items - for example open selected item in text editor or cd into directory.
Take a look at an example of such a config:
`
{
    "Execute": {
        "foo.OpenEditor": [
            "micro $result"
        ]
    },
    "Keymap": {
        "M-e": "foo.OpenEditor"
    }
}
`

**Comment by lestrrat** [comment]
I don't think this adds anything more than what can be done via "$cmd $(peco ...)" or "peco --exec"
